frontend/app/routes/_protected+/_index.tsx

```diff
@@ -41,6 +41,11 @@ export default function Index() {
     <>
       <p className="mb-8 border-b border-gray-200 pb-8 text-lg text-gray-500">{t('index:welcome', { firstName: userInfo.firstName, lastName: userInfo.lastName })}</p>
       <div className="grid gap-4">
+        {useFeature('apply') && (
```

Comment:
In order to make this feature flag configurable, we have to add it as part of the `ENABLED_FEATURES` environment variable. We should update `.env.example` as well as the `validFeatureNames` array in `env.server.ts` to incorporate this new feature flag.

Reply:
Alternatively, we could remove the use of this feature flag (and card) from the `/_protected+/_index.tsx` page. This particular protected index page will only be shown when a user is logged in through MSCA.

All pages within the `/apply` flow do not have authentication and will be accessible outside MSCA.

Reply:
I will removed it (ed62887534ecb1a1d9810f348c88031cfc813d39). Had only added for local testing and had accidentally pushed it.